### DISCUSSION TITLE
The 'Connect-PnPOnline' command was found in the module 'PnP.PowerShell', but the module could not be loaded.

### DISCUSSION THREAD

**Discussion by Sgalbus** [pnp/powershell#1133] [Q&A]
If I execute this command:

$connection=Connect-PnPOnline -Url $oneDriveUrl -Credentials $O365_Credentials -ReturnConnection

I obtain this error:

Connect-PnPOnline : The 'Connect-PnPOnline' command was found in the module 'PnP.PowerShell', but the module could not
be loaded. For more information, run 'Import-Module PnP.PowerShell'.

It seems that the error appers only if I open a remote PS session toward a SP2019 server before executing Connect-PnPOnline. 

These are PS modules installed on the machine:

Version                     Name                                                  Repository
-------                       ----                                                      ----------
2.0.2.129                  AzureADPreview                                 PSGallery
2.0.4                        ExchangeOnlineManagement             PSGallery
16.0.21513.12000    Microsoft.Online.SharePoint.Powe...   PSGallery
2.0.0                        MicrosoftTeams                                   PSGallery
1.1.183.57               MSOnline                                             PSGallery
1.4.7                       PackageManagement                           PSGallery
1.7.0                       PnP.PowerShell                                     PSGallery
2.2.5                       PowerShellGet                                      PSGallery

**Comment by SophievB1** [comment]
Could you solve the problem? I have the same :/
